src/mrpro/operators/Functional.py

```diff
@@ -162,6 +170,12 @@ def prox_convex_conj(self, x: torch.Tensor, sigma: torch.Tensor | float = 1.0) -
         sigma[sigma < 1e-8] += 1e-6
         return (x - sigma * self.prox(x / sigma, 1 / sigma)[0],)
 
+    def __rmul__(self, scalar: torch.Tensor | complex) -> ProximableFunctional:
```

Comment:
is there a reason why scalar is either a tensor or complex? Because in the __rmul__ for Functional is either tensor or float and it made more sense to me

Reply:
in the rmul, I had to add a type ignore because of a pytorch bug. That type ignore also silenced that it should be complex (not float). Not that multiplication with a complex number makes much sense, but because all Operators can be multiplied by complex numbers, and Functionals are Operators.. so you can still multiply them -- just the prox does not give a correct result, but throws an exception.